lib/shopify_api/rest/request.ex

```diff
@@ -197,6 +197,11 @@ defmodule ShopifyAPI.REST.Request do
 
   defp remaining_calls(_), do: nil
 
+  # Absolute URL generator
+  defp url(%{shop_name: domain}, <<?/, path::binary>>),
```

Comment:
Could this instead be `"/" <> path`? I think that's slightly more explicit about what we're matching on.

Reply:
Might be a bit more Elixir-ery, but I wouldn't say more explicit.

```<< >>``` matches on binaries with ```<<?/ , path >>``` matching a binary that starts with "/" and assigning the rest in to path.

Reply:
I understand how it works, but how often do we use this style of binary pattern matching elsewhere in the code base?